workbench-core/swb-common-ui/src/api/users.ts

```diff
@@ -16,7 +16,8 @@ const createUser = async (createUserForm: CreateUserForm): Promise<void> => {
   await httpApiPost(`users`, { ...createUserForm });
 };
 
-const useUsers = () => {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const useUsers = (): any => {
```

Comment:
can we type this?

Reply:
Unfortunately we cannot, the type of users is not defined at this package. Both users and mutate need the `any`

Reply:
Would it be better if the users type/interface be defined here? Since this is the function that gets the users for the application

Reply:
Not fixing for now, we can revisit later. Need to merge this PR to unblock the team.

Reply:
Leave a TODO to revisit later?